index.js

```diff
@@ -114,39 +124,48 @@ export default class TextMarquee extends PureComponent {
   }
 
   animateBounce = () => {
-    const {duration, marqueeDelay, loop, isInteraction, useNativeDriver, easing, children} = this.props
+    const {duration, marqueeDelay, loop, isInteraction, useNativeDriver, easing, children, scrollingSpeed} = this.props
     this.setTimeout(() => {
       Animated.sequence([
         Animated.timing(this.animatedValue, {
           toValue:         -this.distance - 10,
-          duration:        duration || children.length * 50,
+          duration:        duration || (this.distance) * scrollingSpeed,
           easing:          easing,
           isInteraction:   isInteraction,
           useNativeDriver: useNativeDriver
         }),
         Animated.timing(this.animatedValue, {
           toValue:         10,
-          duration:        duration || children.length * 50,
+          duration:        duration || (this.distance) * scrollingSpeed,
           easing:          easing,
           isInteraction:   isInteraction,
           useNativeDriver: useNativeDriver
         })
       ]).start(({finished}) => {
+        if (finished) {
+          this._hasFinishedFirstLoop = true
```

Comment:
Just to follow previous convention and what's already there, can you remove `_` from method names?

Reply:
Have updated this